### DISCUSSION TITLE
Want some project inspiration? Not sure what to work on? Ideas for Computing

### DISCUSSION THREAD

**Discussion by ghost** [community/community#20524] [Discussions]
Hi

I'm a DevOps software engineer interested in fast, scalable, distributed, parallel and concurrent systems.

I write my ideas for computing on GitHub. I use GitHub as my blog. Whenever I think of a computer or software idea I publish it on an ideas repository on my GitHub. When I get to around 100-150 ideas I create a new repository.

Check out my ideas pages.

https://GitHub.com/samsquire/ideas
https://GitHub.com/samsquire/ideas2
https://GitHub.com/samsquire/ideas3
https://GitHub.com/samsquire/ideas4

I also thought of business ideas.
https://GitHub.com/samsquire/startups

I hope this helps someone looking for inspiration.

I would be happy to talk over your ideas and clarify any details. There's a Chinese translation of the first ideas page as linked.

**Comment by radar-blip** [comment]
Good work on taking the time to do this, and sharing with others. I'm sure people will find it helpful! 👍🏼

**Reply by ghost** [reply]
Thank you for your positive feedback. I hope it helps or even inspires people to do the same! Thanks again!

**Reply by radar-blip** [reply]
Anytime. I have starred it and will take another look at it soon. I could use some ideas myself!
Have a great day!

**Comment by GabrielLukecs** [comment]
This is a goldmine of inspiration, Sam. I love the concept of using GitHub as a public "ideas" log—it’s such a practical way to track the evolution of a project before the first line of code is even written.

I’ll definitely be diving into your startups repo; we’re actually building out some similar resources over at [StartupsFlow.com](https://startupsflow.com) to help founders navigate these exact kinds of technical and architectural hurdles. Thanks for putting these out there!
